worker/lib/workers/qemu.ts

```diff
@@ -76,6 +80,30 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 		if (this.screenCapturer != null) {
 			await this.screenCapturer.teardown();
 		}
+
+		// remove iptables rules set up from host
+		try{
+			await execProm(`iptables-legacy-save | grep -v 'comment ${this.iptablesComment}' | iptables-legacy-restore`)
+		} catch (e){
+			console.log(`error while removing iptables rules: ${e}`)
+		}
+
+		try{
+			await execProm(`ip link set dev ${this.qemuOptions.network.bridgeName} down`);
+			await execProm(`brctl delbr ${this.qemuOptions.network.bridgeName}`);
+		} catch(e){
+			console.log(`error while removing bridge: ${e}`)
+		}
+
+		await new Promise((resolve, reject) => {
+			if (this.dnsmasqProc && !this.dnsmasqProc.killed) {
+				// don't return until the process is dead
+				!this.dnsmasqProc.on('exit', resolve)
```

Comment:
What is the `!` here for? Also, missing semicolon.

Reply:
Thats a typo, well spotted